test/unit/codegen/codegen_neuron_cpp_visitor.cpp

```diff
@@ -227,10 +227,7 @@ void _nrn_mechanism_register_data_fields(Args&&... args) {
         THEN("Placeholder registration function is printed") {
             std::string expected_placeholder_reg = R"(/** register channel with the simulator */
     void __test_reg() {
-        /* s */
-        _slist1[0] = {4, 0};
-        /* Ds */
-        _dlist1[0] = {7, 0};
+        _initlists();
```

Comment:
The values inside should be stable. For one or two concrete cases we can look for the string:
```
_slist1[0] = {4, 0}
```
and
```
_dlist1[0] = {7, 0}
```